dev_guide/deployments.adoc

```diff
@@ -670,3 +670,42 @@ $ oc scale dc frontend --replicas=3
 
 The number of replicas eventually propagates to the desired and current
 state of the deployment configured by the deployment configuration `frontend`.
+
+
+[[assigning-pods-to-specific-nodes]]
+
+== Assigning Pods to Specific Nodes
+
+You can use node selectors in conjunction with labeled nodes to control pod placement.
+
+ifdef::openshift-enterprise,openshift-origin[]
+[NOTE]
+====
+OpenShift administrators can assign labels link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+endif::[]
+
+Cluster administrators link:../admin_guide/managing_projects.html#using-node-selectors[can set the default node selector] for your project in order to restrict pod placement to specific nodes. As an OpenShift developer, you can set a node selector on a pod configuration to restrict nodes even further.
+
+To add a node selector when creating a pod, edit the pod configuration, and add the `nodeSelector` value. This can be added to a single pod configuration, or in a pod template:
+
+====
+----
+apiVersion: v1
+kind: Pod
+spec:
+  nodeSelector:
+    disktype: ssd
+...
+...
+----
+====
+
+Pods created when the node selector is in place are assigned to nodes with the specified labels.
+
+The labels specified here are used in conjunction with the labels link:../admin_guide/managing_projects.html#using-node-selectors[added by a cluster administrator]. For example, if a project has the `type=user-node` and `region=east` labels added to a project by the cluster administrator, and you add the above `disktype: ssd` label to a pod, the pod will only ever be scheduled on nodes that have all three labels. 
+
```

Comment:
On Wed, Mar 2, 2016 at 12:34 AM, Timothy Poitras <notifications@github.com>
wrote:
>
> As an OpenShift developer, how can I view the default node selector
> configuration that was configured by the cluster administrator?
>
>
>
I don't believe there is a way at this time. They can see the project
annotation if it exists but not the global default. Although it's easy to
figure out what it is by creating a pod and looking at the node selector it
ends up with. I don't think that's worth mentioning.